_posts/en/podcast/2023-03-02-newsletter-recap.md

```diff
@@ -0,0 +1,412 @@
+signoff of multiple people.  The problem of course is that the device that
+combines the information from the multiple shares does learn, at least
+temporarily, the secret, so the idea is that is some sort of hardware device
+that is trusted to only take the inputs from the users and then not be able to
+export the key.
+
+I know that the Fireblocks I think is MPC-based instead of multisig-based, and
+that way they could use essentially the same solutions for Bitcoin and Ethereum;
+but it also scared some of the cryptographers that I was working with, how new
+the MPC crypto was, and I may have made mistakes here.  It was like three years
+ago that I talked to people about this.
+
+{:#hwi}
+_HWI 2.2.1_
+
+**Mike Schmidt**: So, we have three releases that we covered in the newsletter
+yesterday.  The first one is HWI 2.2.1, which is a maintenance release.  I saw a
+couple of notable things.  One is signing of some P2SH multisigs using the
+Trezor hardware device; and then the second thing was pubkey serialization in
+ranged descriptors and, Murch, I thought it would be useful maybe for you to use
+this opportunity to describe what is a ranged descriptor.
+
+**Mark Erhardt**: A ranged descriptor is basically just a descriptor that
+describes a whole series of scriptpubkeys within one output descriptor.  So,
+output descriptors in general are a construct that's basically an upgraded
+version of an extended pubkey, where you define the general scheme of a
+scriptpubkey and leave wildcards for the key material.  I use a
+Pay-to-Public-Key-Hash construction, but the keys are from this series of
+subkeys in a BIP32 graph.  So, ranged descriptor is basically just like a series
+of scriptpubkeys that have the same scheme, but different pubkeys plugged in.
+
+**Mike Schmidt**: Excellent, thank you for that overview.  So, those were the
+notable things in HWI.
+
+{:#cln}
+_Core Lightning 23.02rc3_
+
+The next notable release candidate is Core Lightning 23.02rc3, which we've had
+in actually for a couple of weeks now.  And so, if you go back I think to our
+Recap for #238, we had a couple of folks from Blockstream in the Core Lightning
+team that came on and gave us an overview of that release candidate, so I think
+you should reference that for more details.
+
+{:#lnd}
+_lnd v0.16.0-beta.rc1_
+
+And then the last release we had is LND v0.16.0 and this is a beta release
+candidate.  A couple of things that I noticed were interesting here was a few
+P2P updates for LND, including a bug fix that could lead to some channel updates
+being missed; and then also some changes at the P2P level for gossip updates.
+They've also added the decryption for larger Onion failure messages, and we've
+covered that a few times in the newsletter.  The ability to have these larger
+error messages can enable better pathfinding using what we've talked with Joost,
+I believe, about fat errors.  So I thought that was interesting.  There was also
```

Comment:
Possibly link to the episode with Joost here. Perhaps linking to the news item meanwhile would be useful? Or do you think the related news item is already sufficiently linked up?

Reply:
Might be enough with the related new item info